Hyper/Texture.cs

```diff
@@ -35,6 +37,48 @@ public static Texture LoadFromFile(string path)
             return new Texture(handle);
         }
 
+        public static Texture LoadFromText(string text)
+        {
+            int handle = GL.GenTexture();
+
+            GL.ActiveTexture(TextureUnit.Texture0);
+            GL.BindTexture(TextureTarget.Texture2D, handle);
+
+            StbImage.stbi_set_flip_vertically_on_load(1);
+
+            using (var bitmap = new SKBitmap(text.Length * 30, 50))
```

Comment:
Can we parameterize this based on font size?

Reply:
We probably could but i thought it would be easier to always have the same size of the texture and just change the size of the HUDElement if we want to resize it.